library/patient.inc

```diff
@@ -1110,7 +1114,7 @@ function updatePatientData($pid, $new, $create = false)
 {
   /*******************************************************************
     $real = getPatientData($pid);
-    $new['DOB'] = fixDate($new['DOB']);
+    $new['DOB'] = DateToYYYYMMDD($new['DOB']);
```

Comment:
yep, this is more complicated than it appears. for example, the CarecoordinationTable.php script uses Y/M/D no matter what.

Reply:
Thus in this case, fixDate is best here to fix the Y/M/D and then the YYYY-DD-MM conversions should happen in scripts.

Reply:
(this is going to be a bit like untangling a large knot :) )

Reply:
TODO
revert above changes and have this instead done in origin scripts

Reply:
This code block was fixed.